server/v2/stf/branch/store.go

```diff
@@ -119,7 +119,7 @@ func (s Store[T]) ChangeSets() (cs []store.ChangeSet, err error) {
 
 	for ; iter.Valid(); iter.Next() {
 		k, v := iter.Key(), iter.Value()
-		cs = append(cs, store.ChangeSet{
+		cs = append(cs, store.StateChange{
```

Comment:
how come account isnt used here? is the change set only for on module?

Reply:
yes this is the single account/module state

Reply:
https://github.com/cosmos/cosmos-sdk/pull/19151/files#diff-338d5aa839ab1345a6e5b1fcfe7d3fd872b1c3b29feaa2ed1b05073023db6de2R19
^ this is the thing that keeps the different branches together